src/data/markdown/docs/01 guides/02 Using k6/01 HTTP requests.md

```diff
@@ -67,8 +67,11 @@ k6 will automatically apply [tags](/using-k6/tags-and-groups#section-tags) to yo
 
 | Name   | Description                                |
 | ------ | ------------------------------------------ |
+| expected_response <sup>(≥ v0.31)</sup>  | By default, response statuses between 200 and 399 are `true`. Change the default behavior with [setResponseCallback](/javascript-api/k6-http/setresponsecallback-callback).                  |
+| group   | When the request runs inside a [group](/javascript-api/k6/group-name-fn), the tag value is the group name.  Default is empty.               |
 | name   | Defaults to URL requested                  |
 | method | Request method (`GET`, `POST`, `PUT` etc.) |
+| scenario   | When the request runs inside a [scenario](/using-k6/scenarios), the tag value is the scenario name.  Default is empty.               |
```

Comment:
Ahh, the default is not empty, the default is `default`. Since k6 v0.27.0, every request runs in a scenario, and every scenario has to have a name.

Reply:
https://github.com/k6io/docs/commit/b4b45cef04b8768bc425290059654ff24f318191